bot/cogs/ext/commands/theotown.py

```diff
@@ -303,6 +306,8 @@ async def suggest(self, ctx: Context, *, suggestion: str = None): # noqa C901
             s_id = await self.api.submit_suggestion(ctx.author.id, message.id, suggestion, attachment_url)
             embed.set_footer(text=f'✅ I like this idea; ❌ I hate this idea; ❗ Already possible.\n#{s_id}')
             await message.edit(embed=embed)
+            if self.use_suggestion_threads:
+                self.client.create_expiring_thread(message, f"{truncate_string(suggestion, 89)} discussion", timedelta_to_datetime(timedelta(days=7)).timestamp())
```

Comment:
I wonder whether this is gonna be a good UX in terms of the thread name

Reply:
That was my main point of concern as well, maybe it would be better to name them according to the suggestion id